src/debug.c

```diff
@@ -1869,15 +1869,20 @@ static void writeStacktraces(int fd, int uplevel) {
     /* ThreadsManager_runOnThreads returns NULL if it is already running */
     if (!stacktraces_data) return;
 
+    size_t skipped = 0;
```

Comment:
i don't see why this (and the output of get_ready_to_signal_threads_tids) needs to be size_t.
size_t is used to count bytes (bound by the addressable memory), but here we count threads, so i think it can be int, or if for some reason we want to handle more than 4G threads on 64 bit system, it can be long.
for all i care it can be signed.

this will make it easier to print later (using `%lu` instead of an awkward `%zu`)

Reply:
AFAIK size_t is the common type for representing sizes and counts of objects, not only bytes.
what's wrong with `%zu`? 🤔

Reply:
I think it's usually used for memory and bytes, not for counting (long or int are used there)
Anyway, nothing is wrong with `%zu` it's just less common.
Do whichever you prefer, as long as the compilation error is resolved.